test/Jenkinsfile-applyTemplate

```diff
@@ -0,0 +1,34 @@
+#!groovy
+@Library(["pipeline-library@master"]) _
+
+node("maven") {
+    stage("SETUP: Create deployment files") {
+        def params = """
+                    NAME=applytemplate
+                    DATABASE_SERVICE_NAME=applytemplate
+                    """
+
+        writeFile file: "params.txt", text: params
+
+        sh "curl -L -o cakephp-mysql-persistent.json https://raw.githubusercontent.com/openshift/origin/v3.11.0/examples/quickstarts/cakephp-mysql-persistent.json"
```

Comment:
Is there a particular reason for using persistence here? Would it suffice with ephemeral version of cake php?